### ISSUE TITLE
try to do stop and down in background

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-agent#211]
to speedup down/stop operation let's try to make it in background.

**Comment by vdelendik** [comment]
down is super fast now. stop as expected slower...
the only nuance user have to press enter one more time in the terminal after executing `zebrunner-farm down/stop` command.
But for the `zebrunner-farm stop/down <udid>` we do everything in foreground as there is no sense to parallel single device command, i.e. no need to press enter in terminal to proceed with the next command.

More mass down/stop also printed echo message just in case in the console:
```
Press <enter> to proceed...
```

changes should eb tested on linux and mac too

**Comment by vdelendik** [comment]
reopening as restart on slow mac os is completely broken.
we init mass stop in background and mass start in parallel which is completely wrong.
let's revert closer as won't fix

**Comment by vdelendik** [comment]
closing as won't fix
